2021-11-15 12:18:33,871 INFO    MainThread:33532 [wandb_setup.py:_flush():71] setting env: {}
2021-11-15 12:18:33,871 INFO    MainThread:33532 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-15 12:18:33,872 INFO    MainThread:33532 [wandb_init.py:_log_setup():357] Logging user logs to D:\_jupyter\kaggle\wandb\run-20211115_121833-wsvwbp8z\logs\debug.log
2021-11-15 12:18:33,872 INFO    MainThread:33532 [wandb_init.py:_log_setup():358] Logging internal logs to D:\_jupyter\kaggle\wandb\run-20211115_121833-wsvwbp8z\logs\debug-internal.log
2021-11-15 12:18:33,872 INFO    MainThread:33532 [wandb_init.py:_jupyter_setup():307] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000025AF0B68D00>
2021-11-15 12:18:33,873 INFO    MainThread:33532 [wandb_init.py:init():390] calling init triggers
2021-11-15 12:18:33,873 INFO    MainThread:33532 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {}
2021-11-15 12:18:33,873 INFO    MainThread:33532 [wandb_init.py:init():435] starting backend
2021-11-15 12:18:33,873 INFO    MainThread:33532 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=spawn, using: spawn
2021-11-15 12:18:33,876 INFO    MainThread:33532 [backend.py:ensure_launched():199] starting backend process...
2021-11-15 12:18:33,966 INFO    MainThread:33532 [backend.py:ensure_launched():204] started backend process with pid: 37204
2021-11-15 12:18:33,967 INFO    MainThread:33532 [wandb_init.py:init():444] backend started and connected
2021-11-15 12:18:33,979 INFO    MainThread:33532 [wandb_run.py:_label_probe_notebook():866] probe notebook
2021-11-15 12:18:33,981 INFO    MainThread:33532 [wandb_run.py:_label_probe_notebook():876] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-11-15 12:18:33,981 INFO    MainThread:33532 [wandb_init.py:init():503] updated telemetry
2021-11-15 12:18:33,997 INFO    MainThread:33532 [wandb_init.py:init():533] communicating current version
2021-11-15 12:18:35,517 INFO    MainThread:33532 [wandb_init.py:init():538] got version response 
2021-11-15 12:18:35,517 INFO    MainThread:33532 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2021-11-15 12:18:35,757 INFO    MainThread:33532 [wandb_init.py:init():576] starting run threads in backend
2021-11-15 12:18:40,855 INFO    MainThread:33532 [wandb_run.py:_console_start():1693] atexit reg
2021-11-15 12:18:40,856 INFO    MainThread:33532 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.WRAP
2021-11-15 12:18:40,856 INFO    MainThread:33532 [wandb_run.py:_redirect():1604] Wrapping output streams.
2021-11-15 12:18:40,857 INFO    MainThread:33532 [wandb_run.py:_redirect():1628] Redirects installed.
2021-11-15 12:18:40,858 INFO    MainThread:33532 [wandb_init.py:init():603] run started, returning control to user process
2021-11-15 12:18:40,859 INFO    MainThread:33532 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-15 12:18:40,859 INFO    MainThread:33532 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-15 12:18:40,876 INFO    MainThread:33532 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-15 12:18:48,907 INFO    MainThread:33532 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-15 12:18:48,907 INFO    MainThread:33532 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-15 12:18:48,922 INFO    MainThread:33532 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-15 12:18:48,926 INFO    MainThread:33532 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-15 12:18:48,926 INFO    MainThread:33532 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-15 12:18:48,936 INFO    MainThread:33532 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-15 12:18:49,127 INFO    MainThread:33532 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-15 12:18:49,127 INFO    MainThread:33532 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-15 12:18:49,133 INFO    MainThread:33532 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-15 12:18:49,178 INFO    MainThread:33532 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-15 12:18:49,178 INFO    MainThread:33532 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-15 12:18:49,194 INFO    MainThread:33532 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-15 12:18:49,195 INFO    MainThread:33532 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-15 12:18:49,195 INFO    MainThread:33532 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-15 12:18:49,209 INFO    MainThread:33532 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-15 12:18:49,209 INFO    MainThread:33532 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-15 12:18:49,210 INFO    MainThread:33532 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-15 12:18:49,224 INFO    MainThread:33532 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-15 12:18:49,224 INFO    MainThread:33532 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-15 12:18:49,224 INFO    MainThread:33532 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-15 12:18:49,238 INFO    MainThread:33532 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-15 12:18:49,238 INFO    MainThread:33532 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-15 12:18:49,238 INFO    MainThread:33532 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-15 12:18:49,259 INFO    MainThread:33532 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-15 12:18:49,261 INFO    MainThread:33532 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-15 12:18:49,261 INFO    MainThread:33532 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-15 12:18:49,273 INFO    MainThread:33532 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-15 12:18:49,275 INFO    MainThread:33532 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-15 12:18:49,275 INFO    MainThread:33532 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-15 12:18:49,285 INFO    MainThread:33532 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-15 12:18:49,286 INFO    MainThread:33532 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-15 12:18:49,286 INFO    MainThread:33532 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-15 12:18:49,298 INFO    MainThread:33532 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-15 12:18:49,308 INFO    MainThread:33532 [wandb_setup.py:_flush():71] setting env: {}
2021-11-15 12:18:49,309 INFO    MainThread:33532 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-15 12:18:49,309 INFO    MainThread:33532 [wandb_init.py:_log_setup():357] Logging user logs to D:\_jupyter\kaggle\wandb\run-20211115_121849-2k8zviqw\logs\debug.log
2021-11-15 12:18:49,309 INFO    MainThread:33532 [wandb_init.py:_log_setup():358] Logging internal logs to D:\_jupyter\kaggle\wandb\run-20211115_121849-2k8zviqw\logs\debug-internal.log
2021-11-15 12:18:49,309 INFO    MainThread:33532 [wandb_init.py:init():390] calling init triggers
2021-11-15 12:18:49,309 INFO    MainThread:33532 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {'seed': 2021, 'epochs': 3, 'model_name': 'roberta-base', 'train_batch_size': 64, 'valid_batch_size': 64, 'max_length': 128, 'learning_rate': 0.0001, 'scheduler': 'CosineAnnealingLR', 'min_lr': 1e-06, 'T_max': 500, 'weight_decay': 1e-06, 'n_fold': 5, 'n_accumulate': 1, 'num_classes': 1, 'margin': 0.5, 'device': device(type='cuda', index=0), 'hash_name': '85p02jdqiqnz', 'tokenizer': PreTrainedTokenizerFast(name_or_path='roberta-base', vocab_size=50265, model_max_len=512, is_fast=True, padding_side='right', special_tokens={'bos_token': '<s>', 'eos_token': '</s>', 'unk_token': '<unk>', 'sep_token': '</s>', 'pad_token': '<pad>', 'cls_token': '<s>', 'mask_token': AddedToken("<mask>", rstrip=False, lstrip=True, single_word=False, normalized=False)}), 'group': '85p02jdqiqnz-Baseline'}
2021-11-15 12:18:49,310 INFO    MainThread:33532 [wandb_init.py:init():410] re-initializing run, found existing run on stack: wsvwbp8z
2021-11-15 12:18:49,311 INFO    MainThread:33532 [wandb_run.py:finish():1347] finishing run bigshanedogg/jigsaw/wsvwbp8z
2021-11-15 12:18:49,312 INFO    MainThread:33532 [jupyter.py:save_history():429] not saving jupyter history
2021-11-15 12:18:49,312 INFO    MainThread:33532 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-15 12:18:49,312 INFO    MainThread:33532 [wandb_init.py:_jupyter_teardown():289] cleaning up jupyter logic
2021-11-15 12:18:49,312 INFO    MainThread:33532 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 0
2021-11-15 12:18:49,312 INFO    MainThread:33532 [wandb_run.py:_restore():1635] restore
2021-11-15 12:18:50,049 INFO    MainThread:33532 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 659
  total_bytes: 659
}

2021-11-15 12:18:50,201 INFO    MainThread:33532 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 659
  total_bytes: 659
}

2021-11-15 12:18:50,438 INFO    MainThread:33532 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 659
  total_bytes: 659
}

2021-11-15 12:18:50,927 INFO    MainThread:33532 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 659
  total_bytes: 15720
}

2021-11-15 12:18:51,043 INFO    MainThread:33532 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 659
  total_bytes: 15720
}

2021-11-15 12:18:51,151 INFO    MainThread:33532 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 659
  total_bytes: 15720
}

2021-11-15 12:18:51,254 INFO    MainThread:33532 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15720
  total_bytes: 15720
}

2021-11-15 12:18:51,359 INFO    MainThread:33532 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15720
  total_bytes: 15720
}

2021-11-15 12:18:51,464 INFO    MainThread:33532 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15720
  total_bytes: 15720
}

2021-11-15 12:18:51,571 INFO    MainThread:33532 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15720
  total_bytes: 15720
}

2021-11-15 12:18:51,677 INFO    MainThread:33532 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15720
  total_bytes: 15720
}

2021-11-15 12:18:51,936 INFO    MainThread:33532 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15720
  total_bytes: 15720
}

2021-11-15 12:18:52,463 INFO    MainThread:33532 [wandb_run.py:_wait_for_finish():1793] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15720
  total_bytes: 15720
}
local_info {
}

2021-11-15 12:18:53,579 INFO    MainThread:33532 [wandb_run.py:_append_files():2061] logging synced files
